tests.py

```diff
@@ -359,7 +365,7 @@ class JSONTestCase(object):
     def test_throttle_GET(self):
         r = self.client.get('/user')
         self.assertMethod('GET')
-        self.assertEqual(r, { 'foo': 'bar' })
+        self.assertEqual(r, {'foo': 'bar'})
```

Comment:
Does this work? I believe self.client.get would return a json body, but I doubt that it converts it to a python dict for you.